tests/unit/src/main/scala/tests/QuickBuild.scala

```diff
@@ -244,23 +246,39 @@ object QuickBuild {
       scalaVersion: String,
       scalaBinaryVersion: String,
       sources: Boolean = false
-  ): List[Path] = {
+  ): List[Path] =
     fetchDependencies(
       dependencies.iterator
         .map(d => toDependency(d, scalaVersion, scalaBinaryVersion))
         .toList,
       sources
     )
-  }
   def fetchDependencies(
-      dependencies: List[s.Dependency],
+      dependencies: List[Dependency],
       sources: Boolean = false
   ): List[Path] = {
-    val settings = new s.Settings()
-      .withDependencies(dependencies)
-      .withClassifiers(if (sources) List("sources", "_") else List())
-    s.CoursierSmall.fetch(settings)
+    val classifiers =
+      if (sources) Set("sources").asJava
+      else Set.empty[String].asJava
```

Comment:
you can delay calling `asJava` until the `withClassifiers(classifiers.asJava)`, making the code more readable

Reply:
Fixed.